rotkehlchen/db/dbhandler.py

```diff
@@ -3421,3 +3422,24 @@ def delete_user_note(self, identifier: int) -> None:
             write_cursor.execute('DELETE FROM user_notes WHERE identifier=?', (identifier,))
             if write_cursor.rowcount == 0:
                 raise InputError(f'User note with identifier {identifier} not found in database')
+
+    def get_nft_mappings(self, identifiers: List[str]) -> Dict[str, dict]:
+        """
+        Given a list of nft identifiers, return a list of nft info (id, name, collection_name)
+        for those identifiers.
+        """
+        result = {}
+        identifiers_query = f'identifier IN ({",".join("?" * len(identifiers))})'
+        with self.conn.read_ctx() as cursor:
+            cursor.execute(
+                f'SELECT identifier, name, collection_name FROM nfts WHERE {identifiers_query}',
+                identifiers,
```

Comment:
That's not what I meant. There is no reason to create new variables. Waste of memory and less readable hunting stuff down. This is what I asked.

```suggestion
                f'SELECT identifier, name, collection_name FROM nfts WHERE '
                f'identifier IN ({",".join("?" * len(identifiers))})'',
                identifiers,
```

Reply:
Aha, I see, alright :+1: